srs_data/sampling.py

```diff
@@ -52,32 +63,58 @@ def _validate_input(self):
         pass
 
     def _sample_production_logs(self):
+        hostname = ""
```

Comment:
Not crazy about this. Would rather let the following code blow up if `hostname` is not `'comcast-kibana.asapp.com'` nor `'sprint-kibana.asapp.com'`. What should happen if `hostname = ""`?

Reply:
yeah you're right, there should be an else that blows it up. although I think it should really part of an overall test in general, where the client name should only be in the expected list. then the remaining code can operate under the assumption that all the configs and input were already checked.

Reply:
Well, don't think you need an `else`: you can just leave `hostname` undefined, and then have the command that uses `hostname` blow up itself.